src/crewai/memory/storage/rag_storage.py

```diff
@@ -150,9 +150,14 @@ def _generate_embedding(self, text: str, metadata: Dict[str, Any]) -> None:  # t
 
     def reset(self) -> None:
         try:
-            shutil.rmtree(f"{db_storage_path()}/{self.type}")
+            # First reset ChromaDB's internal state
             if self.app:
                 self.app.reset()
+            
```

Comment:
@theCyberTech did you mean to add this as well in this PR?

Reply:
No, not sure how that even got there, apologies